packages/migration/migrations/hearth/20230331182109-modify-minio-bucket-policy.js

```diff
@@ -26,7 +24,9 @@ const updatedPolicy = `
     {
       "Effect": "Allow",
       "Principal": {
-        "AWS": "arn:aws:iam::minio-user:${DEFAULT_MINIO_ACCESS_KEY}"
+        "AWS": "arn:aws:iam::minio-user:${
+          process.env.MINIO_ACCESS_KEY || DEFAULT_MINIO_ACCESS_KEY
```

Comment:
@naftis I dont see how these env variables are supplied to the migrations service in docker-compose.  I think that is missing